enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -735,3 +735,54 @@
                     (is (thrown-with-msg? Exception #"Please upgrade"
                                           (cmd/v2-load! dump-dir {}))
                         "throws")))))))))))
+
+(deftest pivot-export-test
+  (testing "Pivot table export and load correctly"
+    (let [card1s (atom nil)]
+      (ts/with-random-dump-dir [dump-dir "serdesv2-"]
+        (ts/with-source-and-dest-dbs
+          (ts/with-source-db
+            (mt/dataset test-data
+              (mt/$ids nil
+                (t2.with-temp/with-temp
+                  [Database   {db-id :id}    {:name "Pivot Database"}
+                   Collection {coll-id :id}  {:name "Pivot Collection"}
+                   Card       card           {:name          "Pivot Card"
+                                              :collection_id coll-id
+                                              :dataset_query {:type     :query
+                                                              :database db-id
+                                                              :query    {:source-table $$checkins
+                                                                         :aggregation  [:sum [:field %checkins.id nil]]
+                                                                         :breakout     [[:field %checkins.user_id nil]]}}
+                                              :visualization_settings
+                                              {:pivot_table.column_split
+                                               {:rows    [[:field %users.name {:base-type    :type/Text
+                                                                               :source-field %checkins.user_id}]]
+                                                :columns [[:field %venues.name {:base-type    :type/Text
+                                                                                :source-field %checkins.venue_id}]]
+                                                :values  [[:aggregation 0]]}
+                                               :column_settings
+                                               {(format "[\"ref\",[\"field\",%s,null]]" %users.name)
+                                                {:pivot_table.column_sort_order "descending"}}}}]
+                  (reset! card1s card)
+                  (-> (serdes/with-cache (into [] (extract/extract {})))
+                      (storage/store! dump-dir))))))
+
+          (ts/with-dest-db
+            (mt/dataset test-data
+              (mt/$ids nil
+                (serdes/with-cache (serdes.load/load-metabase! (ingest/ingest-yaml dump-dir)))
+
+                (let [viz (:visualization_settings
+                           (t2/select-one Card :entity_id (:entity_id @card1s)))]
```

Comment:
Hah, I specifically wrote it in this way so that query is on one line and getter is on the other; but I guess I could even use `select-one-fn` here. :)

Reply:
Both are fine objectively, it's just a little strange to read this on two separate lines, I can't quite articulate why.

In my head, this kind of thing is so familiar, I want to read this in one step, so I want it on one line.
```clojure
(let [viz (:visualization_settings (t2/select-one Card :entity_id (:entity_id @card1s)))] ...
;; or better
(let [viz (t2/select-one-fn :visualization_settings Card :entity_id (:entity_id @card1s)))] ...
;; reads as "get visualization_settings from the card"

(let [viz (:visualization_settings 
           (t2/select-one Card :entity_id (:entity_id @card1s)))
;; reads as "get the card, then get the visualization_settings from the card"
```

Reply:
Personal preferences, I guess. :) I'm used to 80 chars line limit, and to structuring most of the complex actions on line-by-line basis, so that two-line expression looked natural to me. Especially given how long `:visualization_settings` is — it just felt uncluttered to put it on a separate line. :)

Reply:
Yep, makes sense. At metabase we generally go up to 120 chars, don't be afraid to use it

Reply:
It's not about fear, it was me who put 80 chars limit in place, takes time to unlearn old habits. 😁